server/routerlicious/packages/lambdas-driver/src/document-router/documentLambda.ts

```diff
@@ -80,6 +90,7 @@ export class DocumentLambda implements IPartitionLambda {
         // Create or update the DocumentPartition
         let document = this.documents.get(routingKey);
         if (!document) {
+            this.logMessageDetails(boxcar, message, boxcar.documentId, boxcar.tenantId);
```

Comment:
One question to help me understand this condition better: what does it mean here when `!document`? Is it that the document is inactive? And in that case, we want to understand more about the ops triggering it's re-activation?

Reply:
Correct..it means that we have not created the documentlambdas yet..basically logging here will tell us which op triggered the creation of the document lambdas